### PR TITLE
add tests

### PR DESCRIPTION
**Pull request by awirb**
Testing fileUtilities

### CODE REVIEW

File: avaframe/log2Report/generateReport.py [hunk 1]
```diff
@@ -12,6 +12,34 @@
...
+        -----------
+        titleString: str
+            string that shall be added
+        reportDKey: dict
+            dictionary with info for string
+        pfile: file
+        italicFont: bool
+            if True write value in italic
+        flagBreak: bool
+            True break if if conidtion is met
```
Comment (fso42) [inline avaframe/log2Report/generateReport.py:29]: ? Typos?
And I don't get the functionality of this flag

File: avaframe/log2Report/generateReport.py [hunk 2]
```diff
@@ -150,7 +166,7 @@ def writeReport(outDir, reportDictList, cfgFLAGS, plotDict=''):
                 reportD['Simulation Results'].update({'type': 'image'})
 
             # Start writing markdown style report
-            with open(os.path.join(outDir, '%s.md' % reportD['simName']), 'w') as pfile:
+            with open(os.path.join(outDir, '%s.md' % reportD['simName']['name']), 'w') as pfile:
```
Comment (fso42) [inline avaframe/log2Report/generateReport.py:169]: switch to pathlib?

**Review by fso42 — CHANGES_REQUESTED** [review]
Could the logTest.log be shortened considerably?

File: avaframe/log2Report/generateReport.py (hunk 1 above)
Comment (awirb) [reply avaframe/log2Report/generateReport.py:29]: I changed this to: 
flagBreak: bool
            if first item in reportDKey is added to pfile - break

**Review by matthiasto — APPROVED** [review]

**Pull request closed without merging**